YoutubeDownloader/Framework/DialogManager.cs

```diff
@@ -47,7 +47,7 @@ await DialogHost.Show(
         }
     }
 
-    public async Task<string?> PromptSaveFilePathAsync(
+    public static async Task<string?> PromptSaveFilePathAsync(
```

Comment:
```suggestion
    public async Task<string?> PromptSaveFilePathAsync(
```

Please avoid unrelated changes